Dart sass 1.34/version bump (#144)
sass-mq/sass-mq: README.md

        mobileLandscape: 480px
    );

    // If you want to display the currently active breakpoint in the top
    // right corner of your site during development, add the breakpoints
    // to this list, ordered by width. For examples: (mobile, tablet, desktop).
    $breakpoints-shown: (mobile, mobileLandscape, tablet, desktop, wide);

    @use 'path/to/mq' with (
      $breakpoints: $breakpoints,
      $show-breakpoints: $breakpoints-shown,
    );

```
### Notes about `@use` Vs `@import`

> Now when using the `@use` directive you have to change your main about how to reason about vars,
> functions or mixins and how they are now seen by Sass.

Reviewer: ```suggestion
When using the `@use` directive, you have to change your mindset when working with vars,
functions or mixins and how they are now seen by Sass.
```

> Previously with the `@import` statement any var, function, or mixing were exposed in the global scope.
> That means that you could define a var like `$mq-media-type: all` in your main sass file and use
> it anywhere as long as the main file had been loaded previously.

Reviewer: ```suggestion
Previously, with the `@import` statement any var, function, or mixin were exposed in the global scope.
That means that you could define a var like `$mq-media-type: all` in your main sass file and use
it anywhere as long as the main file had been loaded previously.
```

> This was possible because vars functions and mixins were set in the global scope.

Reviewer: ```suggestion
This was possible because vars, functions, and mixins were set in the global scope.
```

> One drawback of this behavior was that we need to ensure not to pollute the global scope

Reviewer: ```suggestion
One drawback of this behaviour was that we needed to ensure not to pollute the global scope
```

with common names or names that may be already taken by any other library.

> To solve this matter we mainly use a prefix in vars, functions, or mixins  in order to avoid
> collapsing names.

Reviewer: ```suggestion
To solve this matter, we mostly used a prefix in vars, functions, or mixins in order to avoid
collapsing names.
```

Now with the new `@use` directive, no var, function, or mixin is placed in global scope, and they are
all scoped within the file.

> That means that we explicitly need to include  the partial file in each file that may use its vars,
> functions or mixins (analog to ES6 import modules).

Reviewer: ```suggestion
That means that we explicitly need to include the partial file in each file that may use its vars,
functions or mixins (similar to ES6 import modules).
```

> So previously we could have a typical setup like this:

Reviewer: ```suggestion
So, previously we could have a typical setup like this:
```

```scss
// main.scss
@import 'mq';
@import 'typography';
@import 'layout';
@include mq($from:tablet) {
    ...
}

...

// typography.scss
@include mq($from:tablet) {
    ...
}

```

> Now you will need to explicit import `_mq.scss` file in each file that needs to use any var, function

Reviewer: ```suggestion
Now, you will need to explicitly import the `_mq.scss` file in each file that needs to use any var, function
```

        mobileLandscape: 480px
    );

    // If you want to display the currently active breakpoint in the top
    // right corner of your site during development, add the breakpoints
    // to this list, ordered by width. For example: (mobile, tablet, desktop).
    $mq-show-breakpoints: (mobile, mobileLandscape, tablet, desktop, wide);

    // If _mq.scss is in your project:
    @use 'path/to/mq';
    // With eyeglass:
    @import 'sass-mq';
    // With webpack (and boilerplates such as create-react-app)

>     @import '~sass-mq';

Author: If you are using webpack and want to have mq imported in your JS file, this will work. 
That's how we do it in our react components.

And if you want to use mq from a Sass file then you should do `@use 'path/to/mq'`
